auth0/v3/authentication/get_token.py

```diff
@@ -164,7 +164,8 @@ def login(self, client_id, client_secret, username, password, scope, realm,
             }
         )
 
-    def refresh_token(self, client_id, client_secret, refresh_token, grant_type='refresh_token'):
+    def refresh_token(self, client_id, client_secret, refresh_token, scope,
```

Comment:
I believe this will be a breaking change for those calling it without named arguments. We should place the new value on the last place.